tensorflow/core/kernels/cudnn_rnn_ops.cc

```diff
@@ -1284,6 +1315,148 @@ TF_CALL_float(REGISTER_GPU);
 TF_CALL_double(REGISTER_GPU);
 #undef REGISTER_GPU
 
+// Run the extended forward operation of the RNN model.
+template <typename T>
+class CudnnRNNForwardOpV3<GPUDevice, T> : public CudnnRNNKernelCommon {
```

Comment:
should it subclass `CudnnRNNForwardOp`? 
It should not do autotune, since it uses unpacked data, which only support standard algorithm now (as of 7.4, please include a comment for that), so we output nothing for the `host_reserved` output.

Reply:
Yes, we don't have to do autotuning over algorithms and only the standard algo is supported for the Ex API.